ZipBuilder/Template/README.md

```diff
@@ -65,7 +65,10 @@ in their modulemaps. If you have disabled the "Link Frameworks Automatically"
 option in your Xcode project/workspace, you will need to add the system
 frameworks and libraries listed in each Firebase framework's
 <Name>.framework/Modules/module.modulemap file to your target's or targets'
-"Link Binary With Libraries" build phase.
+"Link Binary With Libraries" build phase.  Specifically, you may see the error
+'ld: warning: Could not find or use auto-linked framework...' which is an
```

Comment:
Use a back-quote for a better markdown generation.

Reply:
Done.